gulpfile.js

```diff
@@ -129,26 +146,35 @@ gulp.task('build-ci', ['clean'], function (cb) {
 });
 
 gulp.task('build-test', function buildTest(cb) {
-    var config = _.defaultsDeep({}, require('./webpack.config'));
-
-    config.entry = {};
-    // the key in the entry, formed by path.join, determines the target folder structure of the test file
-    config.entry[path.join('request', 'repository', 'request-repository-cache')] = './test/request/repository/request-repository-cache.spec.ts';
-
-    config.output.path = testSettings.output;
-
-    var started = false;
-    function processResult(err, stats) {
-        gutil.log('Webpack\n' + stats.toString(config.log));
-
-        if (!started) {
-            started = true;
-            cb();
+    getTests(function collectAllTests(tests) {
+        var config = _.defaultsDeep({}, require('./webpack.config'));
+        var pathPrefix = path.join(testSettings.input, '/');
```

Comment:
well, here's the thing, it _usually_ works on Windows, but it depends on the context of how it's being used. If it's being used with built-in Node.js libraries, it always works. If it's with 3rd party libraries, it _almost_ always works, but if you use it in anything that does an `exec` or `spawn`, it _never_ works.